cmd/security-bootstrapper/entrypoint-scripts/consul_wait_install.sh

```diff
@@ -91,7 +135,22 @@ if [ "${ENABLE_REGISTRY_ACL}" == "true" ]; then
   setupACL_code=$?
   if [ "${setupACL_code}" -ne 0 ]; then
     echo "$(date) failed to set up Consul ACL"
+  else
+    # kick off the token renewal process
+    echo "$(date) launching vault management token renewal process:"
+    if [ -f "${STAGEGATE_REGISTRY_ACL_SECRETSADMINTOKENPATH}" ]; then
+      mgmt_token=$(jq -r '.auth.client_token' "${STAGEGATE_REGISTRY_ACL_SECRETSADMINTOKENPATH}")
+      parsing_token_ret_code=$?
+      if [ "${parsing_token_ret_code}" -eq 0 ]; then
+        renewToken "${mgmt_token}" &
```

Comment:
How is this running it as a different process?

Reply:
the "&" symbol make it run and fork as a background process and thus it is different from the main process we are running in this entry point.

Reply:
Ahh,  ok. did see that....